src/Fable.Transforms/FSharp2Fable.Util.fs

```diff
@@ -185,8 +185,8 @@ module Helpers =
         else not memb.Accessibility.IsPrivate
 
     let makeRange (r: Range.range) =
-        { start = { line = r.StartLine; column = r.StartColumn+1 }
-          ``end``= { line = r.EndLine; column = r.EndColumn+1 }
+        { start = { line = r.StartLine; column = r.StartColumn }
+          ``end``= { line = r.EndLine; column = r.EndColumn }
```

Comment:
Weird, not sure why I had the impression that columns in Babel location data were 1-based. Thanks for the fix!

Reply:
Ok you already merged it (I guess you wanted this released) I'm still playing around with the test suite on how we can add tests for these kind of issues, but it is harder than I thought....

Reply:
Yes, it's not easy. That's why Fable only has functional tests :) Also, because the actual code generation is done by Babel, I'm afraid that tests checking the output may be fragile (they can break with new Babel versions). For improving the generated code and source maps, I think the best way is what you did: add an easy to quickly check the result of changes in the compiler. So I've added the `npm run build check-sourcemaps` target which integrates your tool with the quicktest: whenever you make changes to Quicktest.fs it will be recompiled with latest browser changes and the sourcemaps viewer will be updated (although you've to refresh the browser yourself). I've already added a couple of improvements. The tool is really useful, thanks again!